backend/uclapi/dashboard/migrations/0018_auto_20210625_0714_squashed_0021_auto_20210627_1527.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.24 on 2021-08-29 12:31
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('dashboard', '0018_auto_20210625_0714'), ('dashboard', '0019_auto_20210626_1811'), ('dashboard', '0020_auto_20210626_1812'), ('dashboard', '0021_auto_20210627_1527')]
+
+    dependencies = [
+        ('dashboard', '0017_webhook_last_success'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='webhook',
+            name='last_success',
+        ),
+    ]
```

Comment:
this file seems to be removing the `last_success` field? should we only have the other migration that adds the field?

Reply:
https://docs.djangoproject.com/en/3.2/topics/migrations/#migration-squashing

Reply:
I'm reading through the migrations and I see no mention of `WebhookTriggerHistory` model even before squashing, despite there being a change to that model. I squashed 18 to 21, then 17 to 18... but it resulted in nothing being applied in migrations...